File: quasar/Quasar.DataTransfer/Vendas/Responses/VendaResponse.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;
using Quasar.DataTransfer.Enderecos.Responses;
using Quasar.DataTransfer.FormasPagamento.Responses;

namespace Quasar.DataTransfer.Vendas.Responses
{
    public class VendaResponse
    {
        public int Codigo { get; set; } 
        public EnderecoResponse Endereco { get; set; }
        public FormaPagamentoResponse FormaPagamento { get; set; }
        public StatusVendaResponse StatusVenda { get; set; }
        public IList<ItemVendaResponse> Itens { get; set; }

    }
}

File: quasar-front/src/app/shared/models/responses/venda.response.ts
import { EnderecoResponse } from './endereco.response';
import { FormasPagamentoResponse } from './formas-pagamento.response';
import { ItemVendaResponse } from './itemVenda.response';
import { StatusVendaResponse } from './statusVenda.response';

export class VendaResponse {
  public codigo: number;
  public endereco: EnderecoResponse;
  public formaPagamento: FormasPagamentoResponse;
  public statusVenda: StatusVendaResponse;
  public itens: ItemVendaResponse[];

  constructor(params: Partial<VendaResponse>) {
    this.codigo = params.codigo || 0;
    this.endereco = params.endereco || new EnderecoResponse({});
    this.formaPagamento = params.formaPagamento || new FormasPagamentoResponse({});
    this.statusVenda = params.statusVenda || new StatusVendaResponse({});
    this.itens = params.itens || [];
  }
}
